### ISSUE TITLE
Design lib interface

### ISSUE THREAD

**Issue by babestvl** [wongnai/tonkatsu-amp#8]

**Comment by PNNutkung** [comment]
#### Default
```
const tonkatsu = new TonkatsuAMP()
```

#### Initiate with config
```
const tonkatsu = new TonkatsuAMP('redis', {
  ...redisConfig
})
```

**Comment by PNNutkung** [comment]
#### Sample Use case
```
const htmlString = `
<iframe width="560" height="315" src="https://www.youtube.com/embed/sAXCxmXi8wA" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
`
tonkatsu.amplify(htmlString)
```

**Comment by PNNutkung** [comment]
> #### Default
> ```js
> const tonkatsu = new TonkatsuAMP()
> ```
> 
> #### Initiate with config
> ```js
> const tonkatsu = new TonkatsuAMP('redis', {
>   ...redisConfig
> })
> ```

https://www.typescriptlang.org/docs/handbook/utility-types.html#recordkt

**Comment by PNNutkung** [comment]
Fix by #17
